src/current/v24.3/follower-reads.md

```diff
@@ -175,9 +173,61 @@ This example performs a bounded staleness follower read against a [demo cluster]
     (7 rows)
     ~~~
 
-### Verify that a cluster is performing follower reads
+### Verify that CockroachDB is performing follower reads
+
+To verify that a cluster is performing follower reads, go to the [**Custom Chart** debug page in the DB Console]({% link {{ page.version.version }}/ui-custom-chart-debug-page.md %}) and add the metric `follower_reads.success_count` to the time-series graph. The number of follower reads performed by your cluster will be shown.
+
+To verify that a specific query uses a follower read, use [`EXPLAIN ANALYZE`]({% link {{ page.version.version }}/explain-analyze.md %}) to see the statement plan. For example, to test the [preceding example](#run-queries-that-use-bounded-staleness-follower-reads):
+
+1. Use the `\demo ls` [shell command]({% link {{ page.version.version }}/cockroach-demo.md %}#commands) to list the connection parameters for all nodes:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    \demo ls
+    ~~~
+
+    The output will list the connection parameters for each node:
+
+    ~~~
+    node 3:
+    ...
+       (sql)      postgresql://demo:demo37199@127.0.0.1:26259/movr?options=-ccluster%3Ddemoapp&sslmode=equire&sslrootcert=%2FUsers%2Fuser%2F.cockroach-demo%2Fca.crt
+    ~~~
+
+1. Open a new terminal and open the SQL shell on a non-leaseholder node. For example: 
```

Comment:
How do you know which is a non-leaseholder node?

Reply:
@yuzefovich I just picked one at random, but is there a straightforward way to tell this? ^

Reply:
@florence-crl I don't think there is an easy way to tell, tbh ... if there is, I will document it in a separate PR!

Reply:
You can query `SELECT lease_holder FROM [SHOW RANGES FROM TABLE promo_codes WITH DETAILS];` to get the leaseholder for all ranges of the table called `promo_codes`.